### ISSUE TITLE
Add option to past list of directories, rather than single `iac-dir`

### ISSUE THREAD

**Issue by ana-cdc** [antonbabenko/pre-commit-terraform#580]
### What problem are you facing?

**TL;DR**: I want to scan all terraform files in my repository, but need to add `--non-recursive`. Suggestion: add possibility to pass list of directories.

 
#### Use case

I have a Terraform repository which currently implements several of `pre-commit-terraform`'s actions. However, I'm having trouble with `terrascan`, given the files my repository contains:
```bash
├── main.tf
├── child-dir
│   ├── example.txt
│   ├── example.sh
│   └── grandchild-dir
│        └── tinyexample.sh
└── child-tf-dir
    └── example.tf
```

I need to run `terrascan --non-recursive` to avoid errors due to `child-dir`, which contains non-`tf` files. But I also need to scan `child-tf-dir`, as it contains `tf` files.

Currently, the only way I found to do this was to add the entire block for `terrascan` a second time.

```yaml
- repo: https://github.com/antonbabenko/pre-commit-terraform/
    rev: v1.83.5
    hooks:
      - id: terrascan
        args:
          - --args=--iac-dir="."
          - --args=--iac-type='terraform'
          - --args=--policy-type=gcp
          - --args=--find-vuln
          - --args=--non-recursive
      - id: terrascan
        args:
          - --args=--iac-dir="./child-tf-dir"
          - --args=--iac-type='terraform'
          - --args=--policy-type=gcp
          - --args=--find-vuln
          - --args=--non-recursive
```

But that means I need to add a new block every time a new folder with `tf` files is made, and that is a level of maintenance overhead I cannot put onto the other devs working on the project.

#### The Request

As such, I'm suggesting the addition of a new argument, `iac-dirs` or `iac-dir-list` or whatever name you devs prefer, which could be passed into the `terrascan` hook.

```yaml
- repo: https://github.com/antonbabenko/pre-commit-terraform/
    rev: v1.83.5
    hooks:
      - id: terrascan
        args:
          - --args=--iac-dir-list=("." "./child-tf-dir")
          - --args=--iac-type='terraform'
          - --args=--policy-type=gcp
          - --args=--find-vuln
          - --args=--non-recursive
```

### How could pre-commit-terraform help solve your problem?

By being able to pass a list of directories, I could:
1. keep my pre-commit code DRY
2. make sure all `.tf` files are checked by `terrascan`, even the future ones
3. reduce the overhead for devs working on this project

### P.S.:

My ultimate goal would actually be to detect the directories containing `.tf` files automatically and only pass this through a variable to `terrascan`.

That is, in fact, what I have done in my GitHub workflow using the `pathlister` action:

```
echo "PATHS=$(echo '${{ steps.pathlister.outputs.paths }}' | sed -e 's:[^/]*${{env.TYPE}}::g' | sed 's: :\n:g' | awk '!a[$0]++' | tr '\n' ' ')" >> $GITHUB_ENV
```

If anyone reading this has tips on how to run a similar command on my `pre-commit-config.yaml`, I would really appreciate tips.

**Comment by yermulnik** [comment]
@ana-cdc Looks like you're looking for `exclude` arg in `pre-commit`: https://pre-commit.com/#pre-commit-configyaml---hooks
![image](https://github.com/antonbabenko/pre-commit-terraform/assets/1274789/afdc0610-d7b6-4238-a77e-e22ba277db04)

See e.g. this example for TF validate: https://github.com/antonbabenko/pre-commit-terraform#terraform_validate
![image](https://github.com/antonbabenko/pre-commit-terraform/assets/1274789/249c2b8f-4b48-4217-8d5b-78c62cf3c40f)

**Comment by ana-cdc** [comment]
Thank you so much! I completely missed the `exclude` argument when looking this up, I'm so sorry for the unnecessary issue created.
